### PR TITLE
Use Facebook Graph API v2.8

### PR DESCRIPTION
**Pull request by bruce-fs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@
 /tmp/
 
 *.gem
+.ruby-version
```
Comment (claudiofullscreen) [inline .gitignore]: @bruce-fs This change seems unrelated to this PR. Can you remove it? Thanks.

Comment (bruce-fs) [inline .gitignore]: @claudiofullscreen removed
